app/services/evss_claim_service.rb

```diff
@@ -39,17 +39,45 @@ def update_from_remote(claim)
   end
 
   def request_decision(claim)
-    EVSS::RequestDecision.perform_async(auth_headers, claim.evss_id)
+    # Workaround for non-Veteran users
+    headers = auth_headers
+    headers_supplemented = false
+
+    # If there is no file number, populate the header another way
+    if headers['va_eauth_birlsfilenumber'].blank?
+      supplement_auth_headers(headers)
+      headers_supplemented = true
+    end
+
+    job_id = EVSS::RequestDecision.perform_async(auth_headers, claim.evss_id)
+
+    record_workaround('request_decision', claim.evss_id, job_id) if headers_supplemented
+
+    job_id
   end
 
   # upload file to s3 and enqueue job to upload to EVSS, used by Claim Status Tool
   # EVSS::DocumentsService is where the uploading of documents actually happens
   def upload_document(evss_claim_document)
     uploader = EVSSClaimDocumentUploader.new(@user.uuid, evss_claim_document.uploader_ids)
     uploader.store!(evss_claim_document.file_obj)
+
     # the uploader sanitizes the filename before storing, so set our doc to match
     evss_claim_document.file_name = uploader.final_filename
-    EVSS::DocumentUpload.perform_async(auth_headers, @user.uuid, evss_claim_document.to_serializable_hash)
+
+    # Workaround for non-Veteran users
+    headers = auth_headers
```

Comment:
Maybe dry up the "Workaround for non-Veteran users" block of code since it's called in both methods.